### PR TITLE
Added missing parameters from ZCS AZZURRO 3PH inverters

### PR DESCRIPTION
**Pull request by kavejo**
I have reviewed the registry values from the file that ZCS shared and have removed comments from sensors that were removed.
I've added translations for Italian language and edited the documentation to better identify rebranded 3-phase Sofar inverters.

This is the file ZCS shared: [ModBus-RTU.Communication.Protocol.3PH.Hybrid.and.Inverter.pdf](https://github.com/StephanJoubert/home_assistant_solarman/files/10862339/ModBus-RTU.Communication.Protocol.3PH.Hybrid.and.Inverter.pdf)

### CODE REVIEW

**Comment by kavejo** [comment]
@StephanJoubert, tagging you to give visibility of this. Please do let me know if you think this could be merged. Thanks!

**Comment by kavejo** [comment]
Tagging also the top contributors with the highest number of commits according to Github Insihgts: @jmccrohan, @YodaDaCoda, @lukasniemeier. Could you please review if this could be merged? Thanks!

File: custom_components/solarman/inverter_definitions/sofar_g3hyd.yaml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Sofar G3 also HYD 5-20KTL-3PH
-# some value commented after month no change
+# This works also for rebranded ZCS Azzurro 3-Phase inverters such as the 3PH HYD6000 ZSS
+# Note that this won't work if your ZCS inverter is connected via Connext, you have to be using a Wi-Fi or Ethernet Kit such as ZSM-WIFI-USB.
+# Defined but commented Battery 2 as that was empty during my tests.
```
Comment (YodaDaCoda) [inline custom_components/solarman/inverter_definitions/sofar_g3hyd.yaml:4]: Do you have two batteries attached to the inverter? If not, that may explain why it's blank. Would be better IMO to surface the values, and let the user choose to ignore them.

Same for Module Temperature 2-3 and Radiator Temperature 2-6

File: custom_components/solarman/translations/it.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (YodaDaCoda) [inline custom_components/solarman/translations/it.json:1]: Would recommend moving this into a separate PR - would be nice to get it merged in the event that this PR takes a while to be approved.

File: custom_components/solarman/inverter_definitions/sofar_g3hyd.yaml (hunk 1 above)
Comment (kavejo) [reply custom_components/solarman/inverter_definitions/sofar_g3hyd.yaml:4]: Sadly not, I only have one single battery pack (1 BMS + 9 Batteries from Pylontech).  Happy to remove the comment so that they get surfaced anyway and then users can decide to hide/ignore them.

File: custom_components/solarman/translations/it.json (hunk 2 above)
Comment (kavejo) [reply custom_components/solarman/translations/it.json:1]: Sure, let me find out how to do that and I'd be happy to have it moved to a separate PR. If you know how to do so and want to give me few pointers, that'd be super welcome. Thanks!

Comment (kavejo) [reply custom_components/solarman/translations/it.json:1]: @YodaDaCoda , please check Pull Request #243. I took the liberty to separate the addition of some comments on the Readme.MD on #244. Hope that can help. Thanks

File: custom_components/solarman/inverter_definitions/sofar_g3hyd.yaml (hunk 1 above)
Comment (YodaDaCoda) [reply custom_components/solarman/inverter_definitions/sofar_g3hyd.yaml:4]: Yeah please surface that information.

File: custom_components/solarman/translations/it.json (hunk 2 above)
Comment (YodaDaCoda) [reply custom_components/solarman/translations/it.json:1]: Can you plz update this PR (make new commits on `main` branch of your fork) to remove this file & changes to the readme as those changes are now covered by #243 & #244?

Comment (kavejo) [reply custom_components/solarman/translations/it.json:1]: I just re-enabled the commented sensors however I have not found a way to remove the other 2 commits from the branch. Do you have any pointer please? Thanks!

Comment (kavejo) [reply custom_components/solarman/translations/it.json:1]: Happy to do like I had done for the readme.md, and just revert origin/main to upstream/main, make the change, and then force-push to my origin/main without touching the 2 files referenced in the other PRs. If that's OK just let me know and I'll proceed.

Comment (YodaDaCoda) [reply custom_components/solarman/translations/it.json:1]: you could pull the files from `upstream/main`, commit the changes, then rebase & squash history. more-or-less how i'd do it anyway.

`git fetch origin main`
`git checkout origin/main -- your-file-here`
`git commit ...`
`git rebase -i origin/main`
